src/exampleTypes/Facet.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import _ from 'lodash/fp'
+import * as F from 'futil-js'
+import { observer } from 'mobx-react'
+import { exampleTypes } from 'contexture-client'
+import { Flex } from '../layout/Flex'
+import injectTreeNode from '../utils/injectTreeNode'
+
+// For futil-js
+let toggleElement = (check, val, arr = []) =>
+  (check ? _.pull : F.push)(val, arr)
+
+export default injectTreeNode(
+  observer(({ tree, node, hide = {}, ...props }) => (
+    <div {...props}>
+      {!hide.facetFilter && (
+        <input
+          className="contexture-search-box"
+          type="text"
+          value={node.optionsFilter}
+          onChange={e =>
+            tree.mutate(node.path, { optionsFilter: e.target.value })
+          }
+          placeholder="Find..."
```

Comment:
This should probably be customizable.

Reply:
That may be true, but these are exampleTypes. While we intend to use them in production, we should wait until there's a real use case for overriding the placeholder since every instance of facet currently has this placeholder in our main application. Example types will eventually be split into their own repo, so their API shouldn't be considered as first class as the core contexture-react components. Also, that's something we can easily add in a minor release but would have to release a major revision to remove it.